pdal/Reader.cpp

```diff
@@ -40,7 +40,7 @@ namespace pdal
 
 void Reader::readerAddArgs(ProgramArgs& args)
 {
-    m_filenameArg = &args.add("filename", "Name of file to read", m_filename);
+    m_filenameArg = &args.addRequired("filename", "Name of file to read", m_filename);
```

Comment:
Just an example of usage, though we can just add params to the current add function.